provider/aws/session.go

```diff
@@ -109,7 +109,7 @@ func newSession(awsConfig AWSSessionConfig) (*session.Session, error) {
 
 	if awsConfig.AssumeRole != "" {
 		if awsConfig.AssumeRoleExternalID != "" {
-			logrus.Infof("Assuming role: %s with external id %s", awsConfig.AssumeRole, awsConfig.AssumeRoleExternalID)
+			logrus.Infof("Assuming role %s with external id", awsConfig.AssumeRole)
```

Comment:
```suggestion
			logrus.Infof("Assuming role %s with external id", awsConfig.AssumeRole)
			logrus.Debug(awsConfig.AssumeRoleExternalID)
```
Since it can be usefull when debugging roles, wdyt of displaying it with debug level ?

Reply:
That's a great idea